packages/babel-plugin-extract-gql/test/unit/fixtures/actual/.4.js

```diff
@@ -0,0 +1,12 @@
+import { useQuery, persist } from 'graphql-hooks'
```

Comment:
why is this file `.4.js` instead `4.js`?

Reply:
To disable this test case. Basically this should cover the case where an external options object is supplied to `useQuery` (e.g. `useQuery(quer, optObj)`). The code doesn't yet know how to manipulate that object.

But simpler cases like `useQuery(query)` OR `useQuery(query, { a: 1 })` are covered and working fine.

I was hoping to come back and fix the 4th test.